HW7/task3.py

```diff
@@ -0,0 +1,13 @@
+def factorial(n):
+    if n == 0 or n == 1:
+        return 1
+    return n * factorial(n - 1)
+
+# I wanted to use n == [0, 1]:
```

Comment:
I think it can be implemented like `if n in [0, 1]`. `n` is a number so it can't be equal to list